pynicotine/gtkgui/userbrowse.py

```diff
@@ -808,36 +804,27 @@ def on_folder_transfer_accelerator(self, *args):
             return True
 
         if len(self.file_store) <= 0:
-            # risk of accidental recursive download
+            # don't risk accidental recursive download
             self.on_folder_expand_sub_accelerator()
             return True
 
         self.on_download_directory()  # without prompt
         return True
 
     def on_folder_open_manager_accelerator(self, *args):
-        """ Ctrl+Alt+Enter: Open folder in File Manager...
-            (or Alt+Enter)  Download Folder Into...    """
-
-        if len(self.file_store) <= 0:
-            self.on_folder_expand_sub_accelerator()
+        """ Ctrl+Alt+Enter: Open folder in File Manager... """
 
         if self.user == config.sections["server"]["login"]:
             self.on_file_manager()
-
-        elif len(self.file_store) >= 1:  # [user is not self]
-            self.on_download_directory_to()
-
-        return True
+            return True
```

Comment:
Move this outside the if clause, or return False by default, depending on what the intention is.

Reply:
Done. I see the sense in it, but in our case return True or False never has a useful effect because the key press event won't cascade once it has already parsed. If the conditions were refactored to allow falling back to happen then maybe the code would be more efficient, but it would make it even more difficult to follow the logic.